integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/logging/CCOMMONS22LoggingAdminTestCase.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.esb.logging;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.log4j.lf5.LogLevel;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.logging.admin.stub.types.carbon.AppenderData;
+import org.wso2.esb.integration.common.clients.logging.LoggingAdminClient;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+
+import static org.testng.Assert.assertEquals;
+
+/*
+ * This test case is testing the following functionality in logging admin.
+ *    1. Restore to defaults functionality (CCOMMONS-22).
+ *    2. Get appender data for an appender name (CCOMMONS-23).
+ */
+public class CCOMMONS22LoggingAdminTestCase extends ESBIntegrationTest {
+    private static final Log log = LogFactory.getLog(CCOMMONS22LoggingAdminTestCase.class);
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        // start the server in super tenant mode
+        super.init();
+    }
+
+    @Test(groups = "wso2.esb", description = "Set the global logging to different level and restore defaults")
+    public void testRestoreDefaults() throws Exception {
```

Comment:
Shall we add a doc comment describing what is tested exactly?

Reply:
what is tested is in test annotation description, if the description depicts what is tested, I think we don't need doc comment